<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input_Score</class>
 <widget class="QDialog" name="Input_Score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Input_Score{background-image: url(:/picture/bgp11.png);}</string>
  </property>
  <widget class="QLabel" name="prompt">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>-5</y>
     <width>791</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Please input the student's ID or name or the
course's ID or name to query the corresponding scores

(You can also query all the studentsâ€™
all courses' scores by inputing nothing)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_stu_id">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="stu_name">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student's Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="stu_id">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>238</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student's ID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>520</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home Page</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_stu_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>287</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="cou_id">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>354</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Course's ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cou_name">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>406</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Course's Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_cou_id">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>346</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_cou_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>404</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
